### ISSUE TITLE
[HTML] Loading resource on relative path returns file not found.

### ISSUE THREAD

**Issue by mscode98** [libgdx/libgdx#6911]
#### Issue details
When exporting the project to html, if we give relative paths to assets, the browser can't load the assets.

#### Reproduction steps/code
1. Create a texture that loads a resource using a relative path.
```java
var texture = new Texture(Gdx.files.internal("./texture.png"));
```
2. Run `./gradlew html:dist` or `./gradlew html:superDev`
3. In your browser devtools the console should complain about resource not found on the texture path.
![image](https://user-images.githubusercontent.com/101365429/175823527-ff385c5a-202a-45f6-9408-3785dbdeccde.png)

#### Version of libGDX and/or relevant dependencies
- LibGDX - 1.11.0

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [x] HTML/GWT
- [ ] Windows
- [x] Linux
- [ ] macOS

**Comment by Frosty-J** [comment]
Interesting issue, though since [`internal()` is already relative](https://libgdx.com/wiki/file-handling#file-storage-types) you can just do `Gdx.files.internal("texture.png")` instead.

**Comment by tommyettinger** [comment]
The code you gave, `var texture = new Texture(Gdx.files.internal("./texture.png"));`, uses `var`, which is only present in Java language level 11 or later. Language level 11 is supported by GWT 2.9.0 (and the just-released 2.10.0), but libGDX currently uses GWT 2.8.2, which only supports up to language level 8. I'll try a superfluous `./` to see if it still breaks, but as @Frosty-J said, it isn't required. You definitely can't do `../` to load outside the current assets folder.

OK, just tried it, and prepending `./` definitely stops internal paths from working on HTML. This is a bug, even if it is fairly minor.

**Comment by mgsx-dev** [comment]
Since internal path can be a classpath (project ressources, jar), relative paths (`./` or `../`) are not always supported and shouldn't be used with internal path.
This limitation has been mentioned in other issues : 
https://github.com/libgdx/libgdx/issues/6125#issuecomment-670184752
https://github.com/libgdx/libgdx/issues/5936#issuecomment-595204559
